specification/dimensions/skupriceid.md

```diff
@@ -0,0 +1,54 @@
+### SKU Price ID
+
+A SKU Price ID is a unique identifier that defines the price of a SKU that incurred a charge. This identifier represents a unique collection of nullable attributes that determine the price. SKU Price ID can be referenced on a price list published by a provider to look up detailed information about the SKU and the unit price associated with the SKU. The composition of the detailed information associated with the price may differ from one provider to the next.
```

Comment:
> A SKU Price ID is a unique identifier that defines the price of a SKU that incurred a charge.

While a charge relates to a single SKU and a single Quantity In Pricing Unit, it contains different costs: List Cost, Negotiated Cost, Effective Cost, Billed Cost.

Thus, I believe some clarity is expected by readers about which Unit Price the SKU Price ID is intended define.
Otherwise, too much room could be left for interpretation.

For example, let's consider the example of a VM usage that has 2 charges, because it is partly covered by a commitment-based discount:
- One charge where the VM usage is priced with commitment-based discount
- One charge where the VM is priced at the on-demand date
In this case, I believe the SKU Price ID would be different in the two charges.

Maybe to make it more precise:
> A SKU Price ID is a unique identifier that defines the price *that is used to bill the SKU in the charge.*

Reply:
I think that's a very fair point!  I will change.